.bazelrc

```diff
@@ -352,7 +355,6 @@ build:clang-cl --copt="-Wno-builtin-macro-redefined"
 #     overrides a member function but is not marked 'override'
 #   MOCK_METHOD(void, addCallbacks, (StreamCallbacks & callbacks));
 build:clang-cl --copt="-Wno-inconsistent-missing-override"
-build:clang-cl --action_env=USE_CLANG_CL=1
```

Comment:
this isn't required any longer?

Reply:
Yes it's still required, and occured 2x in .bazelrc

Reply:
See first occurace at line 341

Reply:
Ah gotcha!